fnmatch.go

```diff
@@ -0,0 +1,169 @@
+package editorconfig
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func FnmatchCase(pattern, name string) bool {
+	p, _ := translate(pattern)
+	r, err := regexp.Compile(fmt.Sprintf("^%s$", p))
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Oops. An error here definitely shouldn't exit the app. It should actually be returned.